src/Mod/PartDesign/App/ShapeBinder.cpp

```diff
@@ -612,13 +747,23 @@ void SubShapeBinder::slotRecomputedObject(const App::DocumentObject& Obj) {
     if(Context.getValue() == &Obj
         && !this->testStatus(App::ObjectStatus::Recompute2))
     {
-        update();
+        try {
+            update();
+        } catch (Base::Exception &e) {
+            e.ReportException();
+        }
     }
 }
 
+static const char _GroupPrefix[] = "Configuration (";

```

Comment:
This causes a compiler warning: unused variable _GroupPrefix [-Wunused-const-variable]
This is possibly a copy&paste issue since it's also declared in Link.cpp